lib/collection/src/common/snapshots_manager.rs

```diff
@@ -1,5 +1,7 @@
+use std::env;
 use std::path::Path;
 
+use aws_sdk_s3::Client as S3Client;
```

Comment:
Have you considered other crates? Like e.g. https://crates.io/crates/object_store which seems to be x5 smaller and doesn't have that strong affiliation with AWS

Reply:
Definitely worht trying with this!